src/stream/src/executor/over_window/eowc.rs

```diff
@@ -200,11 +200,7 @@ impl<S: StateStore> EowcOverWindowExecutor<S> {
         // Recover states from state table.
         let table_iter = this
             .state_table
-            .iter_with_prefix(
-                partition_key,
-                sub_range,
-                PrefetchOptions::new_for_exhaust_iter(),
```

Comment:
Why disable prefetch here?

Reply:
Because the new prefetch algorithm may cause more IO bandwidth. I think we shall use it more cautiously.